src/components/Form.js

```diff
@@ -20,31 +20,37 @@ export default function Form({
   } = useForm({
     defaultValues: preloadedValues
       ? preloadedValues
-      : { activities: '', friend: '', notes: '', date: '', location: '' },
+      : {
+          activities: '',
+          friend: '',
+          notes: '',
+          date: '',
+          location: '',
+        },
   });
 
   const onSubmit = data => {
     if (preloadedValues) {
       handleActivity({
         id: preloadedValues.id,
         activity: data.activity,
-        friend: data.friend,
+        friend: data.friend === '' ? 'I still need to plan...' : data.friend,
```

Comment:
Thank you - but it should be in there in case when editing the friends name will be taken out, there should then automatically filled in this text in order to categorize it on the front page properly😊

Reply:
As we discussed yesterday, you can do that in a different way with conditional rendering :) hope it helps :)